code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

```diff
@@ -333,13 +333,39 @@
 			M.AdjustWeakened(2)
 	..()
 
+	if(reac_volume > 11)
+		metabolization_rate = 1 * REAGENTS_METABOLISM
+
+		if(istype(M, /mob/living/carbon/human))
```

Comment:
In fact I just noticed this is a copy of the code from the overdose_process(). Is that not called automatically if someone goes over the threshold for any given reagent?